pyproject.toml

```diff
@@ -122,6 +122,46 @@ requires = ["poetry-core>=1.1.0"]
 build-backend = "poetry.core.masonry.api"
 
 
+[tool.ruff]
+fix = true
+target-version = "py37"
+line-length = 88
+extend-select = [
+    "B",  # flake8-bugbear
+    "ANN",  # flake8-annotations
```

Comment:
suggestion, don't bother with `ANN`

by the time you've removed all the lints that disagree with mypy... might just as well use mypy!

Reply:
Agreed 👍 Since we are using the `strict` mode of `mypy`, `flake8-annotations` doesn't add any value anymore.

Reply:
removed